Shared/Samples/Edit feature attachments/README.metadata.json

```
@@ -0,0 +1,35 @@
+{
+    "category": "Edit and Manage Data",
+    "description": "Add, delete, and download attachments for features from a service.",
+    "ignore": false,
+    "images": [
+        "edit-feature-attachments.png"
+    ],
+    "keywords": [
+        "Edit and manage data",
+        "JPEG",
+        "PDF",
+        "PNG",
+        "TXT",
+        "image",
+        "picture",
+        "ArcGISFeature.deleteAttachment(_:)",
+        "Attachments",
+        "FeatureLayer",
+        "ServiceFeatureTable",
+        "ServiceFeatureTable.applyEdits()"
+    ],
+    "redirect_from": [],
+    "relevant_apis": [
+        "ArcGISFeature.deleteAttachment(_:)",
+        "Attachments",
+        "FeatureLayer",
+        "ServiceFeatureTable",
+        "ServiceFeatureTable.applyEdits()"
+    ],
+    "snippets": [
+        "EditFeatureAttachmentsView.Model.swift",
+        "EditFeatureAttachmentsView.swift"
```

The order matters here - the view's file name needs to be placed at the top so that the project can generate the intermediate file correctly.
```suggestion
        "EditFeatureAttachmentsView.swift",
        "EditFeatureAttachmentsView.Model.swift"
```